taxi_service/settings.py

```diff
@@ -56,7 +57,7 @@
 TEMPLATES = [
     {
         "BACKEND": "django.template.backends.django.DjangoTemplates",
-        "DIRS": [],
+        "DIRS": [os.path.join(BASE_DIR, "taxi/../templates")],
```

Comment:
The path to the templates seems to be hard-coded. Consider using os.path.join(BASE_DIR, 'templates') for better portability.